src/sidebar/components/test/share-annotations-panel-test.js

```diff
@@ -0,0 +1,209 @@
+'use strict';
+
+const { createElement } = require('preact');
+const { shallow } = require('enzyme');
+const unroll = require('../../../shared/test/util').unroll;
```

Comment:
`unroll` is effectively deprecated, in favor of using plain JS like this:

```js
[{
  description: 'first case'
  paramA: value,
  paramB: value,
}].forEach(({ paramA, param B }) => {
  // nb. Destructuring in previous line is optional, but helps to catch unused params.
  it(`does something (${description})`, () => {
    // ...
  });
});
```

The rationale is that `unroll` is something extra to learn and doesn't provide a lot of added value on the above.